python/pyspark/sql/dataframe.py

```diff
@@ -889,6 +889,24 @@ def cov(self, col1, col2):
             raise ValueError("col2 should be a string.")
         return self._jdf.stat().cov(col1, col2)
 
+    def freqItems(self, cols, support=1e-2):
```

Comment:
it's probably best to use None as the default, and set support to 1e-2 if it is None in the body of the function. Then you don't need to set the default value in two places.